assets/js/sections/satellite.js

```diff
@@ -108,7 +108,7 @@ function editSatelliteDialog(id) {
 
 function saveUpdatedSatellite(form) {
 	$(".alert").remove();
-	if (form.nameInput.value == "") {
+	if (form.displayNameInput.value == "") {
 		$('#edit_satellite_dialog').prepend('<div class="alert alert-danger" role="alert"><a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>Please enter a name!</div>');
```

Comment:
This `&times;` produces a "x" in front of the alert-text. There's no spacing. Maybe add one?!

And: It has no effect in chrome (perhaps i am using it wrong - from reading the code i assume it should close the message?!)

Same at L59

Reply:
Added a space. The "X" closes the popup (on my Firefox system).